test/examples-test.js

```diff
@@ -28,6 +38,12 @@ process.on('unhandledRejection', (error) => {
     return
   }
 
+  if (error.code === 403) {
+    // when API rate limit is reached 403 Forbidden is thrown
+    // Known files: addCollaborator.js, getContent.js
+    return
+  }
```

Comment:
the rate limit error is actually a tricky one, I run into it myself today. I'll think about ways to work around it without adding more code to the examples

Reply:
@Ethan-Arrowood I've pushed a change that should help with the rate limit errors, see 84dbe00. [Create a new token](https://github.com/settings/tokens/new), it does not need any access. Then create a `.env` file and put the token into it

```
GH_TOKEN=<your token here>
```

Then all requests from the examples are authenticated by default which bumps the rate limit for 60/minute to 6000/minute.

I'll need this finish up soon, if you can work on it today/tomorrow that's great, I can finish up the rest on Wednesday. Let me know if you have any questions :)